spec/features/subscribe_back_spec.rb

```diff
@@ -27,10 +34,29 @@ def given_there_is_content_i_can_subscribe_to
     )
   end
 
+  def when_i_confirm_my_email
+    visit "/email/subscriptions/verify?topic_id=#{@topic_id}?frequency=1&address=user@domain.uk"
+  end
+
   def when_i_start_signup_via_govuk
     visit "/email/subscriptions/new?topic_id=#{@topic_id}"
   end
 
+  def and_i_select_frequency_and_submit
+    stub_email_alert_api_sends_subscription_verification_email("person@somewhere.uk", "daily", @topic_id)
+    choose(option: "daily")
+    within "form.checklist-email-signup" do
```

Comment:
Minor: could we just do `click_on "Continue"` or something? I guess this approach might be more robust, but on the other hand, I had to think more about what the code is doing.